### PR TITLE
Sort-last draw support

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: livre/core/data/NodeId.cpp [hunk 1]
```diff
@@ -30,30 +30,26 @@ NodeId::NodeId( const Identifier& identifier )
     : _id( identifier )
 {}
 
-NodeId::NodeId( const uint32_t level,
-                const Vector3ui& position,
+NodeId::NodeId( const uint32_t level, const Vector3ui& position,
```
Comment (bilgili) [inline livre/core/data/NodeId.cpp]: I prefer the other way around. It is consistent in code ;) When it comes to style can we use each others ?

File: livre/core/dash/DashRenderNode.h [hunk 2]
```diff
@@ -70,7 +70,7 @@ class DashRenderNode
      * @return True, if object is visible.
      * @warning This condition is set from outside of the object.
      */
-    LIVRECORE_API bool isLODVisible() const;
```
Comment (bilgili) [inline livre/core/dash/DashRenderNode.h]: I clearly remember this function name has been changed on purpose. It was isVisible() before but the functionality is really the LOD is visible.

File: livre/core/data/NodeId.cpp (hunk 1 above)
Comment (bilgili) [inline livre/core/data/NodeId.cpp]: And, I am not seeing any functional change on this file. Good to clean but does it match with keep changes small ?

File: livre/lib/types.h [hunk 3]
```diff
@@ -66,7 +66,10 @@ typedef boost::scoped_ptr< TextureDataCache > TextureDataCachePtr;
 /** Map definitions */
 typedef boost::unordered_map< uint32_t, DataUploadProcessorPtr > DataUploadProcessorPtrMap;
 
-const uint32_t CONNECTION_ID( 0u );
+static const uint32_t CONNECTION_ID( 0u );
+
+typedef std::array< float, 2 > Range;
```
Comment (bilgili) [inline livre/lib/types.h]: Vector2f ? why array ?

File: livre/core/dash/DashRenderNode.h (hunk 2 above)
Comment (eile) [inline livre/core/dash/DashRenderNode.h]: But then it's inconsistent with setVisible(). Change both to set/isLODVisible?

File: livre/core/data/NodeId.cpp (hunk 1 above)
Comment (eile) [inline livre/core/data/NodeId.cpp]: It's in a separate cleanup commit. review the commits individually if you want small changes.

File: livre/lib/types.h (hunk 3 above)
Comment (eile) [inline livre/lib/types.h]: Ok, same difference. Was approaching it from a float[2] angle and thought array is safer. Which one do you prefer?

File: livre/core/dash/DashRenderNode.h (hunk 2 above)
Comment (bilgili) [inline livre/core/dash/DashRenderNode.h]: ok.

File: livre/core/data/NodeId.cpp (hunk 1 above)
Comment (bilgili) [inline livre/core/data/NodeId.cpp]: The reviews can be per commit, I did not know that.

**Comment by eile** [comment]
ping

**Pull request merged**